test/test_torch.py

```diff
@@ -1208,6 +1208,26 @@ def test_dtype_out_match(self):
         d = torch.autograd.Variable(torch.DoubleTensor(2, 3))
         self.assertRaises(RuntimeError, lambda: torch.zeros((2, 3), out=d, dtype=torch.float32))
 
+    def test_make_subclass(self):
+        class SubTensor(torch.Tensor):
+            pass
```

Comment:
I think it's important to also test what happens to member variables on the class, as this is a very important part of the input contract.